templates/ceph/what-is-ceph.html

```diff
@@ -1,282 +1,254 @@
-    <br />
-  </div>
-  <div class="row">
-    <div class="row">
-      <div class="col-10 u-sv3">
-        {{
-          image (
-          url="https://assets.ubuntu.com/v1/b05b5a3c-Ceph_diagrams-02.svg",
-          alt="",
-          height="318",
-          width="661",
-          hi_def=True,
-          loading="lazy"
-          ) | safe
-        }}
-      </div>
+  <section class="p-section">
+    <div class="u-fixed-width p-section--shallow">
+      <hr class="p-rule" />
+      <h2>What is a Ceph cluster?</h2>
     </div>
     <div class="row">
-      <div class="col-8">
-        <p>
-          A Ceph storage cluster consists of the following types of daemons:
-        </p>
-        <ul class="p-list">
-          <li class="p-list__item">
-            Cluster monitors (<code>ceph-mon</code>) that maintain the map of the cluster state, keeping track of active and failed cluster nodes, cluster configuration, and information about data placement and manage authentication.
-          </li>
-          <li class="p-list__item">
-            Managers (<code>ceph-mgr</code>) that maintain cluster runtime metrics, enable dashboarding capabilities, and provide an interface to external monitoring systems.
-          </li>
-          <li class="p-list__item">
-            Object storage devices (<code>ceph-osd</code>) that store data in the Ceph cluster and handle data replication, erasure coding, recovery, and rebalancing. Conceptually, an OSD can be thought of as a slice of CPU/RAM and the underlying SSD or HDD.
-          </li>
-          <li class="p-list__item">
-            Rados Gateways (<code>ceph-rgw</code>) that provide object storage APIs (swift and S3) via http/https.
-          </li>
-          <li class="p-list__item">
-            Metadata servers (<code>ceph-mds</code>) that store metadata for the Ceph File System, mapping filenames and directories of the file system to RADOS objects and enabling the use of POSIX semantics to access the files.
-          </li>
-          <li class="p-list__item">
-            iSCSI Gateways (<code>ceph-iscsi</code>) that provide iSCSI targets for traditional block storage workloads such as VMware or Windows Server.
-          </li>
-        </ul>
-        <p>
-          Ceph stores data as objects within logical storage pools. A Ceph cluster can have multiple pools, each tuned to different performance or capacity use cases. In order to efficiently scale and handle rebalancing and recovery, Ceph shards the pools into placement groups (PGs). The CRUSH algorithm defines the placement group for storing an object and thereafter calculates which Ceph OSDs should store the placement group.
-        </p>
+      <div class="col-9 col-medium-6 col-start-large-4">
+        <div class="p-section--shallow u-hide--small">
+          <div class="p-image-container is-highlighted">
+            {{ image(url="https://assets.ubuntu.com/v1/581ff61d-ceph-chart-2.png",
+                        alt="",
```

Comment:
Is there an alt text available for the chart here? Maybe UX can weigh in on this

Reply:
Thanks both! Question is out to the PM on the copydoc here FYI - will ping when it's updated https://docs.google.com/document/d/1gRuOAg6ZFBp-ikMq-DQBgQ4-BrXGeWOOe2cBxD3ohjE/edit?tab=t.0

Reply:
Just got confirmation on the alt text @immortalcodes! They're added in brackets under the images in the copydoc.